src/commands/env.rs

```diff
@@ -72,7 +74,7 @@ impl Command for Env {
         println!("{}", shell.path(&binary_path)?);
         println!(
             "{}",
-            shell.set_env_var("FNM_MULTISHELL_PATH", multishell_path.to_str().unwrap())
+            shell.set_env_var("FNM_MULTISHELL_PATH", &format_path(multishell_path.to_str().unwrap(), &shell.to_string()))
```

Comment:
Can we avoid stringifying the shells?
We can add a `shell.format_path()` if we want to 👍

Reply:
The fact that every shell is a different struct is weird, I will fix that in the future. This should be 100% enum.